subset/network/network_tests.py

```diff
@@ -0,0 +1,104 @@
+import subprocess
+import time
+import sys
+
+arguments = sys.argv
+
+test_request = str(arguments[1])
+cap_pcap_file = str(arguments[2])
+device_address = str(arguments[3])
+if len(arguments) < 3:
```

Comment:
@ewRedstone If the length of arguments is less that 3 it's going to be difficult to access arguments[4] -> suspected < / > typo.

Could you add in a comment to say why you're doing this? I presume it's for out-of-DAQ testing. Another way to document it could be 

```
a_boolean_with_a_descriptive_name = len(arguments) > 3

if a_boolean_with_a_descriptive_name:
    module_config = str(arguments[4])
    infastructure_exclude_file = str(arguments[5])
```

or

```
standalone_mode = len(arguments) > 3

if standalone_mode:
    module_config = str(arguments[4])
    infastructure_exclude_file = str(arguments[5])
```

Reply:
resolved as suggested